website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,446 @@
+</figure>
+
+#### Support for `position: 'static'`
+
+:::info
+`position: 'static'` is supported only in the New Architecture.
+:::
+
+Elements marked as [`position: 'static'`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#static) may not be offset, and are not considered when determining the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) of an absolutely positioned element. This allows positioning an element relative to an ancestor which is not its direct parent.
+
+<table>
+<tr>
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row-reverse',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 100,
+      height: 100,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: '25%',
+        left: 25,
+        top: 25,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+<td width="300">
+
+![Static Example](../static/blog/assets/0.74-static-example.png)
+
+</td>
+</tr>
+</table>
+
+Notice how the green `<View>` declares `left` and `top` and it is positioned relative to the blue `<View>`, not its parent.
+
+`position: 'static'` may also affect [z-ordering](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) of views. Similar to how `position: 'static'` works with containing blocks, static will not be considered when forming stacking contexts. This allows for stacking an element relative to an ancestor that is not a direct parent.
+
+<table>
+<tr>
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'orange',
+      width: 50,
+      height: 50,
+      left: 10,
+      position: 'relative',
+      zIndex: 10,
+    }}
+  />
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 50,
+      height: 50,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: 25,
+        zIndex: 11,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+<td width="300">
+
+![Static zIndex Example](../static/blog/assets/0.74-zindex-example.png)
+
+</td>
+</tr>
+</table>
+
+React Native continues to default to `position: 'relative'` when no `position` is set.
+
+### New Architecture: Bridgeless by Default
+
+In this release, we are making **Bridgeless Mode the default when the New Architecture is enabled**. You can learn more about our switch to Bridgeless as the default in this post (_coming soon_). To make the transition smoother we enhanced the interop layers to cover Bridgeless and worked with several libraries to make sure they will work in Bridgeless from day one.
```

Comment:
Also, will the post that is linked here be live before this post goes out? Or should that be removed?

Reply:
Very soon, says @cipolleschi, so I'm inclined to keep this placeholder.